doc/examples/perception_pipeline/perception_pipeline_tutorial.rst

```diff
@@ -1,198 +1,96 @@
-:moveit1:
-
-..
-   Once updated for MoveIt 2, remove all lines above title (including this comment and :moveit1: tag)
-
 Perception Pipeline Tutorial
-============================
+==================================
+
+Introduction
+------------
+MoveIt serves sensor plugins for utilizing 3D sensor data in planning pipeline. When it's ran the demo of this tutorial, it should be seen the following rviz screen.
 
-MoveIt allows for seamless integration of 3D sensors using `Octomap <http://octomap.github.io/>`_.
-Once properly configured, you should see something like this in rviz:
+.. raw:: html
 
-.. image:: perception_configuration_demo.png
-   :width: 700px
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/bpMHFxcbRXg" title="MoveIt Perception Pipeline Demo" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Comment:
Very nice that you recorded a video! Instead of linking a youtube video here, wouldn't a gif work too?
Additional, I think it would be nice if the video just contains the relevant video and not your entire screen. I use `kazam` to record videos and it is very straight forward to capture a window only